listenbrainz/webserver/static/js/jsx/follow-users.jsx

```diff
@@ -42,6 +46,40 @@ export class FollowUsers extends React.Component {
     });
   }
 
+  saveFollowList(event) {
+    var listName = this.state.listName;
+    if (this.nameInput.value != undefined && this.nameInput.value != "") {
+      listName = this.nameInput.value;
+    }
+    fetch(this.state.saveUrl, {
+      method: "POST",
+      body: JSON.stringify({
+        "users": this.state.users,
+        "name": listName,
+        "id": this.state.listId,
+      }),
+    })
+    .then(response => response.json())
+    .then(data => {
+      console.debug(data);
+      console.debug("old List ID: " + this.state.listId);
+      this.setState(prevState => {
+        return {listId: data.list_id};
+      });
+      console.debug("new List ID: " + this.state.listId);
+    })
```

Comment:
You also want to `.catch` any error with the promise, and do something with the error. I'd say just write it to console.error for now, I want to come up with a good way to show errors for this component.